packages/components/organisms/f-checkout/src/components/_tests/Checkout.test.js

```diff
@@ -1252,30 +1252,25 @@ describe('Checkout', () => {
     });
 
     describe('watch ::', () => {
-        describe('fulfilmentTimes ::', () => {
+        describe('authToken ::', () => {
             afterEach(() => {
                 jest.clearAllMocks();
             });
 
-            it('should call `selectionChanged` with the first fulfilment time when there are fulfilment times', async () => {
-                // Arrange
-                const setAuthTokenSpy = jest.spyOn(VueCheckout.methods, 'setAuthToken');
+            it('should call `initialise`', async () => {
+            // Arrange
+                const initialiseSpy = jest.spyOn(VueCheckout.methods, 'initialise');
 
-                const wrapper = shallowMount(VueCheckout, {
+                // Act
+                shallowMount(VueCheckout, {
```

Comment:
Is this testing the watcher or the mounted hook?

Reply:
Good catch. I copy-pasted a test and didn't update it properly. Thanks, @xander-marjoram. I've sorted it out now.